lib/buffer.js

```diff
@@ -500,7 +500,8 @@ Buffer.prototype.equals = function equals(b) {
 };
 
 
-// Inspect
+// Override how buffers are presented by util.inspect().
+Buffer.prototype[internalUtil.inspectSymbol] =
 Buffer.prototype.inspect = function inspect() {
```

Comment:
Just a nit but... Perhaps instead:

```js
Buffer.prototype[internalUtil.inspectSymbol] = function inspect() { ... };
Buffer.prototype.inspect = Buffer.prototype[internalUtil.inspectSymbol];
```

I know they accomplish the same thing, but this would make it a bit more straightforward to deprecate and eventually remove the `Buffer.prototype.inspect` variant later on.